lib/EditableList/EditableListForm.js

```diff
@@ -402,26 +402,27 @@ class EditableListForm extends React.Component {
 
     if (status[item.rowIndex].editing) {
       return (
-        <div style={{ display: 'flex' }}>
-          <Button
-            disabled={pristine || submitting || invalid}
-            marginBottom0
-            id={`clickable-save-${this.testingId}-${item.rowIndex}`}
-            onClick={() => this.onSave(fields, item.rowIndex)}
-            {...(typeof actionProps.save === 'function' ? actionProps.save(item) : {})}
-          >
-            Save
-          </Button>
+        <Layout className="flex">
           <Button
             data-type-button="cancel"
             marginBottom0
             id={`clickable-cancel-${this.testingId}-${item.rowIndex}`}
             onClick={() => this.onCancel(fields, item.rowIndex)}
             {...(typeof actionProps.cancel === 'function' ? actionProps.cancel(item) : {})}
           >
-            Cancel
+            <FormattedMessage id="stripes-smart-components.cancel" />
+          </Button>
+          <Button
+            buttonStyle="primary"
+            disabled={pristine || submitting || invalid}
+            marginBottom0
+            id={`clickable-save-${this.testingId}-${item.rowIndex}`}
+            onClick={() => this.onSave(fields, item.rowIndex)}
+            {...(typeof actionProps.save === 'function' ? actionProps.save(item) : {})}
+          >
+            <FormattedMessage id="stripes-smart-components.save" />
```

Comment:
Use `stripes-core.button.save` and `stripes-core.button.cancel` instead of defining these values locally. True, this problem is already epidemic, but let's not make it worse :)

Reply:
@zburke Ah! Thanks – I'll make the change.

We might want to make a page with a complete overview of universal" translations somewhere. Maybe in the Storybook. I made this for another project: https://framework.share-vde.dev/?path=/story/utils-translations--available-translations

Reply:
Good thinking! I have an open story to consolidate our i18n documentation; perhaps this could be a part of that.